app/services/jobs/compute_flights_job.rb

```diff
@@ -8,6 +8,7 @@ class ComputeFlightsJob < Struct.new(:contest)
   def perform
     @contest = contest
     say "Computing flights for #{@contest.year_name}"
+    CategorySynthesisService.synthesize_categories(@contest)
```

Comment:
Does this mean there's a set of synthetic categories for every contest?

Reply:
No.